app/views/partials/foyer/ressources/montants.html

```diff
@@ -59,12 +124,11 @@ <h3 class="panel-title"><span class="short-label">{{ individuVM.label }}</span>
       </div>
     </div>
   </div>
-</div>
 
-<div ng-repeat="ressource in individuVM.ressources | filter: isRessourceOtherTns">
-  <div class="panel panel-montant tns">
+  <div class="panel panel-montant tns"
+   ng-if="ressource.type.id === 'autresRevenusTns'">
```

Comment:
Either keep the current heavyweight yoda-style + triple-equal, or switch to a simpler double-equal, var-first style.